templates/email/quote-expired.hypr

```diff
@@ -0,0 +1,26 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+         <!--- Header --->
+            <div class="mz-curbside-email-header">  
+            <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+            <div class="mz-curbside-header-body">
+                <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+            </div>
+            </div>
+           
+<br />
+<div>
+{% if model.IsSeller %}
+<br /><br />
+   {{ labels.quoteExpiredSeller|string_format(model.name, model.number)|safe }} {{ model.expirationDate|date('M d, Y') }}. {{ labels.pleaseReview }} <a href="//{{ domainName }}/selleraccount/quote/{{model.id}}">{{labels.hereLink}}</a>.
```

Comment:
This link won't work for sellers. We have to link to the seller domain and not the storefront website. Need to figure out the right URL and update this.

This applies to the other emails as well. Need to fix all of them.

Reply:
Changed url: 
{% if model.IsSeller %}
     <a href="//{{ domainName }}/Admin/s-{{model.siteId}}/b2baccounts/edit/{{model.customerAccountId}}">{{labels.hereLink}}</a>.
    {% else %}
     <a href="//{{ domainName }}/myaccount/quote/{{model.id}}">{{labels.hereLink}}</a>.
    {% endif %}